rfcs/ambassador-program/index.md

```diff
@@ -0,0 +1,83 @@
+* Increased GraphQL content online and off
+* Community building - increase the number of “known faces” within GraphQL, increased approachability (addressing the concern of “lowering the barrier to join an inclusive GraphQL community” raised by the Community WG)  
+
+How can these goals be measured? 
+
+* Engagement metrics - social, chat channels, search, downloads, page views…
+* New community members (across social media comments, Discord users, local gathering attendees, working group members…) 
+* Volume of talks at conferences
+* Volume of articles written
+
+### Ambassador Tasks
+
+As a GraphQL Ambassador, you will be an important touchstone for people interested in GraphQL to learn more about it in a friendly and constructive way. 
+
+One or more of the following:
+
+* Conference & meetup talks on GraphQL
+* Local gathering organization
+* Written GraphQL content
+* Video / audio GraphQL content
+
+### Foundation Tasks
+
+* Grant available for travel expenses for approved conference & meetup talks
+* Microgrant (or honorarium) available for content creation 
+    * Where is this content hosted? 
+* Promotion of Ambassador material - conference talk videos and created content
+* Promotion of the Ambassador as a go-to GraphQL expert
+    * Website profile
+    * Longer task: biographies / interviews with each Ambassador
+    * Credly badge
+* Exclusive GraphQL swag eg Ambassador hoodie
+
+### Ambassador Requirements
+
+* Highly knowledgeable about GraphQL and readily able to articulate those ideas 
+* Technical in tone, without sounding like they are on a sales team
+* Be a model community member: Adhering to the code of conduct especially when critical of GraphQL or defending it to others criticism, remaining respectful, constructive and fair 
+* Do not come across to other community members as selling another product, service or agenda
+
+
+### Ambassador Code of Conduct
+
+While the wider GraphQL community abides by the [contributor covenant code of conduct](https://graphql.org/codeofconduct/), we will need a stronger set of standards for the Ambassador Program. This is because the Foundation will be endorsing these people as representatives. We can look to the [CNCF Ambassador Program Standards of Excellence](https://www.cncf.io/people/ambassadors/program-standards/) as a good indicator of what this should be. In fact, we are encouraged to borrow liberally from them, as they are also a Linux Foundation project with an already successful program, and their lawyers have already signed it off. 
+
+### Process
+
+Yearly program
+ * 1 year term? Or mirror the TSC and use a 2 year term  
+
+How many Ambassadors is a reasonable number?  
+
+Self-nominations and nominations from within the community; who should review these?  
+
+Process for removing an Ambassador? And the reason to do so?  
+
+Content generated will need to be reviewed before being promoted  
```

Comment:
I agree on reviewing content before promotion on official foundation channels. However, I'm not sure about reviewing content (such as slides and video material) produced for conferences - even when we promote that a GraphQL talk is happening at such conference. This is an immense workload for us and might discourage frequent submissions. We should be confident in the ambassadors to understand our code of conduct and represent the values and technical direction the TSC is taking, as we do with conference talks at GraphQL conf.

Reply:
You add a valid point about reviewing conference material and I think that would be overreach of the program.

Perhaps minimal review is needed, if we're trusting someone to do a conference talk then we should also be able to trust their written content. I feel like written content might be more open to being a vendor pitch? I'm not sure. Perhaps we don't promote their content if it's eg on their employer's platform.

Reply:
Can the review be left to the discretion of whoever handles the social handles? They will be trusted to do what's best for the community.

Since this is all very public, anyone can give feedback if a submission crosses the line and the "GraphQL community manager" can amend the editorial line.

Reply:
OK, outsource the social promotion to the social media managers sounds good. In practice, at the moment, that is Jory (Linux Foundation) and Benjie who both will also be heavily involved in the Ambassadors program I imagine. But yes, this makes sense.